apps/sharedtest/test/unit/elements/gaia_grid/grid_dragdrop_test.js

```diff
@@ -100,6 +100,7 @@ suite('GaiaGrid > DragDrop', function() {
       stopImmediatePropagation: function() {},
       preventDefault: function() {}
     });
+    this.sinon.clock.tick(20);
```

Comment:
nit: the ```20``` makes me squirm a bit as I think it would be fairly easy to forget where this is coming from in the future. I might opt to export that value on GridDragDrop, then access that in the test. I think that would be more clear, but if you just want to add a comment or something that might work as well.